samples/compute/README.md

```diff
@@ -0,0 +1,66 @@
+---
+page_type: sample
+languages:
+- typescript
+products:
+- azure
+description: "These code samples will show you how to manage Compute using Azure SDK for Typescript."
+urlFragment: compute
```

Comment:
Can we double check this? 
languages:
javascript or typescript?

Reply:
Actually, it can support multple language configuration, but these samples themself are written by typescipt. So I was't sure whether we need to add javascript here.

Reply:
could you double check data-plane Track 2 JS samples?

Reply:
as long as it's consistent